code/__DEFINES/traits.dm

```diff
@@ -582,7 +582,7 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 #define HIGHLANDER_TRAIT "highlander"
 
 ///generic atom traits
-/// Trait from [/datum/component/rust]. Its rusty and should be applying a special overlay to denote this.
+/// Trait from [/datum/element/rust]. Its rusty and should be applying a special overlay to denote this.
```

Comment:
Why is this diff showing? Did you mess up a merge?

Reply:
no clue man, I'll do a check to see if I missed anything tho

Reply:
Turns out the guy who made rust into an element never actually updated this. So I'm right master is wrong?